core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/blueprints/BlueprintToBlueprintV4RequestConverter.java

```diff
@@ -9,10 +9,10 @@
 import com.sequenceiq.cloudbreak.domain.Blueprint;
 
 @Component
-public class BlueprintToBlueprintRequestConverter
+public class BlueprintToBlueprintV4RequestConverter
         extends AbstractConversionServiceAwareConverter<Blueprint, BlueprintV4Request> {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(BlueprintToBlueprintRequestConverter.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(BlueprintToBlueprintV4RequestConverter.class);
 
     @Override
     public BlueprintV4Request convert(Blueprint source) {
```

Comment:
We can send the URL via request, we should give back this.

Reply:
url is not saved in the entity so cannot be retrieved here